### PR TITLE
PMM-7 Remove dev guide for checks v1

### PR DESCRIPTION
**Pull request by artemgavrilov**

### CODE REVIEW

File: docs/details/develop-checks/index.md [hunk 1]
```diff
@@ -32,20 +32,245 @@ PMM uses Alertmanager API to get information about failed checks and show them o
...
+Note that check development in PMM is currently for **debugging only** and **NOT for production use!**  Future releases plan to include the option to run custom local checks in addition to hosted Percona Platform checks.
+
+To develop custom checks for PMM:
+
+1. Install the latest PMM Server and PMM Client builds following the [installation instructions](https://www.percona.com/software/pmm/quickstart#).
+2. Run PMM Server with special environment variables:
+
+   - `PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml` to use checks from the local files instead of downloading them from Percona Platform.
+   - `PERCONA_TEST_CHECKS_DISABLE_START_DELAY=true` to disable the default check execution start delay. This is currently set to one minute, so that checks run upon system start.
+   - `PERCONA_TEST_CHECKS_RESEND_INTERVAL=2s` to define the frequency for sending the SA-based alerts to Alertmanager.
```
[the remark this answers is not in the document]
Comment (artemgavrilov) [reply docs/details/develop-checks/index.md]: Good catch! Fixed.
My IDE renders markdown differently.

File: docs/details/develop-checks/index.md [hunk 2]
```diff
@@ -32,20 +32,245 @@ PMM uses Alertmanager API to get information about failed checks and show them o
...
+    ![!](../../_images/FailedChecks.png)
+
+8.  Go into the Docker container to output the logs of pmm-managed and read the check logs:
+
+```sh
+# get inside the container
+docker exec -it pmm-server bash
+# print and watch the logs
+supervisorctl tail -f pmm-managed
+```
```
Comment (atymchuk) [inline docs/details/develop-checks/index.md:262]:
```suggestion
    ```sh
    # get inside the container
    docker exec -it pmm-server bash
    # print and watch the logs
    supervisorctl tail -f pmm-managed
    ```
```

Comment (atymchuk) [reply docs/details/develop-checks/index.md:262]: I noticed that p.8 is misaligned compared to the rest of code examples:
Screenshot 2023-05-03 at 11 05 24

Comment (atymchuk) [reply docs/details/develop-checks/index.md:262]: I wonder if the same can be achieved with just one command, i.e.:
`docker exec -it pmm-server supervisorctl tail -f pmm-managed`?

Comment (artemgavrilov) [reply docs/details/develop-checks/index.md]: Done

**Review by atymchuk — APPROVED** [review]

**Review by catalinaadam — APPROVED** [review]

**Pull request merged**